src/main/java/org/broadinstitute/hellbender/CommandLineProgramTest.java

```diff
@@ -97,6 +98,11 @@ public static void addDataflowRunnerArgs(ArgumentsBuilder args) {
             args.add("--runner");
             args.add(DataflowCommandLineProgram.getRunnerTypeName(dataflowRunner));
         }
+        String dataflowRunnerEnv = System.getenv(ENV_DATAFLOW_RUNNER);
```

Comment:
It seems like this code is getting a bit redundant.  Can you add a function `getExternallySpecifiedRunner` that checks both the property and the env variable and returns the appropriate value.  Then refactor to use that in these two functions and in `GatkTestPipeline`.  That way we can eliminate the duplicated boilerplate.